client/src/components/Settings/SettingsCanvasesView.res

```diff
@@ -1,5 +1,8 @@
 open Tc
 
+// WHAT is the new 'system' for when there should/shouldn't be newlines between
+// modules here?
```

Comment:
Standard ones first: Tc, Base types, prelude

Then our modules.

The 3rd party modules.

That's generally how it's been done so far